deploy/deploy_to_bintray.bat

```diff
@@ -0,0 +1,93 @@
+rem @echo off
+
+if not defined bintray_deploy (
+    set bintray_deploy=False
+)
+if %bintray_deploy% == False (
+    goto :EOF
+)
+
+if not defined BINTRAY_API_KEY (
+    goto :EOF
+)
+if [%BINTRAY_API_KEY%] == [] (
+    goto :EOF
+)
+
+set API=https://api.bintray.com
+if not defined BINTRAY_USER (
+    set BINTRAY_USER=ChristianBeer
+)
+if not defined BINTRAY_REPO (
+    set BINTRAY_REPO=boinc-ci
+)
+rem owner and user not always the same
+if not defined BINTRAY_REPO_OWNER (
+    set BINTRAY_REPO_OWNER=boinc
+)
+if not defined WEBSITE_URL (
+    set WEBSITE_URL=https://boinc.berkeley.edu
+)
+if not defined ISSUE_TRACKER_URL (
+    set ISSUE_TRACKER_URL=https://github.com/BOINC/boinc/issues
+)
+rem Mandatory for packages in free Bintray repos
+if not defined VCS_URL (
+    set VCS_URL=https://github.com/BOINC/boinc.git
+) 
+set CURL=curl -u%BINTRAY_USER%:%BINTRAY_API_KEY% -H Accept:application/json -w \n
+rem use this for local debugging
+rem set CURL=echo
+
+if not defined pkg_name (
+    set pkg_name=custom
+)
+if not defined pkg_desc (
+    set pkg_desc=Automated CI build of BOINC components
+)
+
+echo Creating package %pkg_name%...
+set data={\"name\": \"%pkg_name%\", \"desc\": \"%pkg_desc%\", \"desc_url\": \"auto\", \"website_url\": [\"%WEBSITE_URL%\"], \"vcs_url\": [\"%VCS_URL%\"], \"issue_tracker_url\": [\"%ISSUE_TRACKER_URL%\"], \"licenses\": [\"LGPL-3.0\"]}
+%CURL% -H Content-Type:application/json -X POST -d "%data%" "%API%/packages/%BINTRAY_REPO_OWNER%/%BINTRAY_REPO%"
+
+for /f "tokens=2-4 delims=/ "  %%a in ("%date%") do (set MM=%%a& set DD=%%b& set YYYY=%%c)
+if not defined build_date (
+    set build_date=%YYYY%-%MM%-%DD%
+)
+if not defined git_rev (
+    set git_rev=%APPVEYOR_REPO_COMMIT:~0,8%
```

Comment:
Please keep this and the next two blocks as they are now. If this is a PR or a scheduled build the variables should already be set correctly.